src/lib/POWERMGNT/POWERMGNT.h

```diff
@@ -15,16 +15,17 @@
 #include <nvs.h>
 #endif
 
-#if defined(TARGET_RX)
+#if defined(HighPower) && !defined(UNLOCK_HIGHER_POWER)
```

Comment:
I think this was wrong before and still is!
Shouldn't this be
```cpp
#if defined(HighPower) && defined(UNLOCK_HIGHER_POWER)
```

Reply:
it is confusing but this seems correct. HighPower is not the max Power, so if it is not unlocked, then maxPower will only get as high as HighPower?

Reply:
> it is confusing but this seems correct. HighPower is not the max Power, so if it is not unlocked, then maxPower will only get as high as HighPower?

Yep very confusing, `UNLOCK_HIGHER_POWER` unlocks `MaxPower` if `HighPower` is defined.